### PR TITLE
feat: Removes tooltip in favor of explains

### PR DESCRIPTION
**Pull request by manueltanzi-okta**
### Description

- Removes tooltip in favor of explains
- Don't show explains when they are equal as label

### Video

explain-top-widget

### Reviewers

@haishengwu-okta @robertjd

### CODE REVIEW

**Comment by manueltanzi-okta** [comment]
@nbarbettini Also take a look! As we talked, we don't show explains unless they are different from labels.

**Comment by nbarbettini** [comment]
Need a review from:
* @maggielaw-okta 
* UI Guild

**Comment by nbarbettini** [comment]
Looks good to the PM. 😄

File: src/views/primary-auth/PrimaryAuthForm.js [hunk 1]
```diff
@@ -126,19 +125,21 @@ define([
...
-          innerTooltip: {
-            title: Okta.loc('primaryauth.password.placeholder', 'login'),
-            text: Okta.loc('primaryauth.password.tooltip', 'login')
-          }
-        }
+        type: 'password'
       };
       if (this.settings.get('features.showPasswordToggleOnSignInPage')) {
-        passwordFieldObject.params.iconDivider = true;
+        passwordFieldObject.params = {};
```
Comment (haishengwu-okta) [inline src/views/primary-auth/PrimaryAuthForm.js]: why this change?

**Comment by alexdahl-okta** [comment]
Looks great to me! I will defer to Maggie on anything.

There's no change to how a developer configures things here, correct? They just provide the same i18n strings?

(String keys like `primaryauth.username.tooltip` are now inaccurate, but I assume renaming them is a breaking change we'd like to avoid. So, no complaint from me.)

**Comment by maggielaw-okta** [comment]
👍from me! Looks really good. 

Out of curiosity, will this change result in the SIW playing nice with the form fill icon button that LastPass inserts? To illustrate what I mean, I've attached a video of me clicking until I eventually find the tiny segment of hotspot that isn't deadened by the tooltip.

siwlastpass

(Cc: @edburyenegren-okta for good measure.)
